david_goodedit/src/App.css

```diff
@@ -3,6 +3,11 @@
   padding: 0;
 }
 
+.darkApp {
+  color: #dddddd !important;
+  background-color: #222222 !important;
```

Comment:
do you know css variables?

https://dev.to/ananyaneogi/create-a-dark-light-mode-switch-with-css-variables-34l8